docs/source-pytorch/common/trainer.rst

```diff
@@ -1231,27 +1231,26 @@ Properties
 callback_metrics
 ****************
 
-The metrics available to callbacks. These are automatically set when you log via `self.log`
+.. autoproperty:: lightning.pytorch.trainer.Trainer.callback_metrics
+   :noindex:
 
-.. code-block:: python
+logged_metrics
+**************
 
-    def training_step(self, batch, batch_idx):
-        self.log("a_val", 2)
+.. autoproperty:: lightning.pytorch.trainer.Trainer.logged_metrics
+   :noindex:
 
+progress_bar_metrics
+********************
 
-    callback_metrics = trainer.callback_metrics
-    assert callback_metrics["a_val"] == 2
+.. autoproperty:: lightning.pytorch.trainer.Trainer.progress_bar_metrics
+   :noindex:
 
 current_epoch
 *************
 
-The number of epochs run.
-
-.. code-block:: python
-
-    if trainer.current_epoch >= 10:
-        ...
-
```

Comment:
Why are we removing the code examples?

Reply:
The examples worth keeping were moved to the docstrings.

This one in particular seems quite silly to me. It's not particularly informative.

Reply:
All the examples removed in this PR are not informative. Otherwise they were kept in the docstring. None of these properties have long examples but `estimated_stepping_batches`, which was kept in the docs